Kernel/Graphics/Console/GenericFramebufferConsole.cpp

```diff
@@ -263,20 +263,15 @@ void GenericFramebufferConsoleImpl::clear(size_t x, size_t y, size_t length)
             if (y >= max_row())
                 y = 0;
         }
-        auto offset_in_framebuffer = framebuffer_offset(x, y);
-        for (size_t glyph_row = 0; glyph_row < m_glyph_rows; glyph_row++) {
-            memset(offset_in_framebuffer.pixels, 0, m_glyph_columns * sizeof(u32));
-            offset_in_framebuffer.bytes += framebuffer_pitch();
-        }
-        flush_glyph(x, y);
```

Comment:
Removing the call of `flush_glyph` might prevent consoles that rely on constant flushing from knowing that the screen is dirty and thus needed to be flushed.

Reply:
I know it looks like I'm removing the call but I'm just replacing the snippet of code there with a call to `clear_glyph(x,y)` which had the same logic, and it calls `flush_glyph(x,y)` before it returns.